### PR TITLE
Fixes reported errors due to multiprocessing on certain OSes

### LINKED ISSUE #31: Nose Timer doesn't work with multiprocess
In order to run my tests faster, I use the multiprocess module and run my tests in parallel, like so:

```
$ nosetests ... --processes=4
```

When I try to run my tests with Nose Timer, I don't see any timer results:

```
$ nosetests ... --processes=4 --with-timer
```

It works fine when I drop "--processes=4"

### LINKED ISSUE #68: v0.5 is broken on MacOs
https://docs.python.org/2/library/multiprocessing.html#multiprocessing.Queue.qsize
```
Note that this may raise NotImplementedError on Unix platforms like Mac OS X where sem_getvalue() is not implemented.
```

I got
```
  File "/Users/user/Projects/vermilion/venv/lib/python2.7/site-packages/nosetimer/plugin.py", line 96, in report
    for i in range(_results_queue.qsize()):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/multiprocessing/queues.py", line 143, in qsize
    return self._maxsize - self._sem._semlock._get_value()
```

It used to work perfectly with multiprocessing and xunitmp with mac, centos and ubuntu, now it fails totally and have to roll back to 0.4.4

### PR DESCRIPTION
**Pull request by mahmoudimus**
* Fixes #68 
* Fixes #31 

Confirmed fixed by https://github.com/mahmoudimus/nose-timer/issues/68#issuecomment-182459804

### CODE REVIEW

File: nosetimer/plugin.py [hunk 1]
```diff
@@ -7,15 +7,62 @@
...
+if not IS_NT:
     import multiprocessing
-    try:
-        import Queue
-    except ImportError:
-        import queue as Queue
-    _results_queue = multiprocessing.Queue()
+    from multiprocessing import queues
+
+    print(multiprocessing.Queue)
```
Comment (mahmoudimus) [inline nosetimer/plugin.py]: @skudriashev should we get rid of this print statement?

File: nosetimer/plugin.py [hunk 2]
```diff
@@ -7,15 +7,62 @@
...
+
+    class TimerQueue(queues.Queue):
```
Comment (mahmoudimus) [inline nosetimer/plugin.py]: Should we move this to a different module?

File: nosetimer/plugin.py [hunk 3]
```diff
@@ -7,15 +7,62 @@
...
 from nose.plugins import Plugin
 
+# try to import Queue
+try:
+    import Queue
+except ImportError:
+    import queue as Queue
+
+# define constants
+IS_NT = os.name == 'nt'
```
Comment (mahmoudimus) [inline nosetimer/plugin.py]: :+1: Great change.

**Comment by mahmoudimus** [comment]
IMO we should get rid of py30->py32 since those are deprecated.

**Comment by skudriashev** [comment]
@mahmoudimus, if you approve this I will submit to master (with fixes) tomorrow and will create new release version. What will you say?

**Comment by mahmoudimus** [comment]
@skudriashev :+1: :shipit: !

**Comment by mahmoudimus** [comment]
@skudriashev thank you! :+1: solid work!

**Pull request merged**